firestore-shorten-urls-bitly/extension.yaml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 name: firestore-shorten-urls-bitly
-displayName: Shorten URLs
+displayName: Shorten URLs with bit.ly
```

Comment:
Don't change the display name in this PR, as that requires marketing review. (We'll do that review shortly before releasing the FDL version)